credential-exchange-types/src/format.rs

```diff
@@ -101,8 +101,12 @@ pub struct LinkedItem {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
-pub struct Extension {
-    pub name: String,
+#[serde(tag = "name", rename_all = "kebab-case")]
+pub enum Extension<E = ()> {
+    #[serde(untagged)]
+    External(E),
+    #[serde(untagged)]
+    Unknown(serde_json::Value),
```

Comment:
`External` would be user defined extensions? Would we just add more members for [the built in extensions](https://fidoalliance.org/specs/cx/cxf-v1.0-wd-20241003.html#sctn-defined-extensions)?